Source/WebKit/Configurations/WebKit.xcconfig

```diff
@@ -236,7 +236,11 @@ WK_EXCLUDED_WEBPUSHD_SANDBOX = $(WK_EXCLUDED_WEBPUSHD_SANDBOX_$(WK_RELOCATABLE_W
 WK_EXCLUDED_WEBPUSHD_SANDBOX_NO = com.apple.WebKit.webpushd.relocatable.mac.sb
 WK_EXCLUDED_WEBPUSHD_SANDBOX_YES = com.apple.WebKit.webpushd.mac.sb
 
-EXCLUDED_SOURCE_FILE_NAMES = $(EXCLUDED_PRODUCT_DEPENDENCY_NAMES) $(EXCLUDED_IOS_RESOURCE_FILE_NAMES) $(EXCLUDED_MACOS_PLUGIN_FILE_NAMES) $(EXCLUDED_MIGRATED_HEADERS_COCOA_TOUCH_$(WK_IS_COCOA_TOUCH)) $(WK_EXCLUDED_WEBPUSHD_SANDBOX);
+EXCLUDED_EXTENSION_SOURCE_FILE_NAMES = Shared/AuxiliaryProcessExtensions/*;
+EXCLUDED_EXTENSION_SOURCE_FILE_NAMES[sdk=iphoneos17.4.internal] = ;
+EXCLUDED_EXTENSION_SOURCE_FILE_NAMES[sdk=iphonesimulator17.4.internal] = ;
```

Comment:
I'd recommend inverting this and adding some extra boilerplate to prevent this from being immediately out of date in future iOS versions.

```suggestion
EXCLUDED_EXTENSION_SOURCE_FILE_NAMES = Shared/AuxiliaryProcessExtensions/*;

// already defined further down…
INCLUDED_SOURCE_FILE_NAMES = $(INCLUDED_EXTENSION_SOURCE_FILE_NAMES_$(USE_INTERNAL_SDK));

INCLUDED_EXTENSION_SOURCE_FILE_NAMES_YES[sdk=iphone*] = Shared/AuxiliaryProcessExtensions/*;
INCLUDED_EXTENSION_SOURCE_FILE_NAMES_YES[sdk=iphone*17.0.internal] = ;
INCLUDED_EXTENSION_SOURCE_FILE_NAMES_YES[sdk=iphone*17.1.internal] = ;
INCLUDED_EXTENSION_SOURCE_FILE_NAMES_YES[sdk=iphone*17.2.internal] = ;
INCLUDED_EXTENSION_SOURCE_FILE_NAMES_YES[sdk=iphone*17.3.internal] = ;```
```

Reply:
Ah, that's a good point. I have updated the patch with this change.

Thanks for reviewing!

Reply:
Sorry, I didn't mean for you to write "already defined further down…" literally. There is already a `INCLUDED_SOURCE_FILE_NAMES` definition in the file, so this change needs to be incorporated to it.

Reply:
Good point, I realized that when I built locally :)

Thanks for reviewing!